api/Models/DB/appdbContext.cs

```diff
@@ -0,0 +1,2211 @@
+    {
+        public appdbContext()
+        {
+        }
+
+        public appdbContext(DbContextOptions<appdbContext> options)
+            : base(options)
+        {
+        }
+
+        public virtual DbSet<Assignment> Assignment { get; set; }
+        public virtual DbSet<AudSheriffDuty> AudSheriffDuty { get; set; }
+        public virtual DbSet<AuthApiScope> AuthApiScope { get; set; }
+        public virtual DbSet<AuthFrontendScope> AuthFrontendScope { get; set; }
+        public virtual DbSet<AuthFrontendScopeApi> AuthFrontendScopeApi { get; set; }
+        public virtual DbSet<AuthFrontendScopePermission> AuthFrontendScopePermission { get; set; }
+        public virtual DbSet<AuthRole> AuthRole { get; set; }
+        public virtual DbSet<AuthRoleApiScope> AuthRoleApiScope { get; set; }
+        public virtual DbSet<AuthRoleFrontendScope> AuthRoleFrontendScope { get; set; }
+        public virtual DbSet<AuthRolePermission> AuthRolePermission { get; set; }
+        public virtual DbSet<AuthUser> AuthUser { get; set; }
+        public virtual DbSet<AuthUserRole> AuthUserRole { get; set; }
+        public virtual DbSet<CourtRoleCode> CourtRoleCode { get; set; }
+        public virtual DbSet<Courtroom> Courtroom { get; set; }
+        public virtual DbSet<Databasechangelog> Databasechangelog { get; set; }
+        public virtual DbSet<Databasechangeloglock> Databasechangeloglock { get; set; }
+        public virtual DbSet<Duty> Duty { get; set; }
+        public virtual DbSet<DutyRecurrence> DutyRecurrence { get; set; }
+        public virtual DbSet<EscortRun> EscortRun { get; set; }
+        public virtual DbSet<GenderCode> GenderCode { get; set; }
+        public virtual DbSet<JailRoleCode> JailRoleCode { get; set; }
+        public virtual DbSet<Leave> Leave { get; set; }
+        public virtual DbSet<LeaveCancelReasonCode> LeaveCancelReasonCode { get; set; }
+        public virtual DbSet<LeaveCode> LeaveCode { get; set; }
+        public virtual DbSet<LeaveSubCode> LeaveSubCode { get; set; }
+        public virtual DbSet<Location> Location { get; set; }
+        public virtual DbSet<OtherAssignCode> OtherAssignCode { get; set; }
+        public virtual DbSet<Region> Region { get; set; }
+        public virtual DbSet<Sheriff> Sheriff { get; set; }
+        public virtual DbSet<SheriffDuty> SheriffDuty { get; set; }
+        public virtual DbSet<SheriffLocation> SheriffLocation { get; set; }
+        public virtual DbSet<SheriffRankCode> SheriffRankCode { get; set; }
+        public virtual DbSet<Shift> Shift { get; set; }
+        public virtual DbSet<WorkSectionCode> WorkSectionCode { get; set; }
+
+        protected override void OnConfiguring(DbContextOptionsBuilder optionsBuilder)
+        {
+            if (!optionsBuilder.IsConfigured)
+            {
+                optionsBuilder.UseNpgsql("Name=ConnectionStrings.DB");
+            }
+        }
+
+        protected override void OnModelCreating(ModelBuilder modelBuilder)
```

Comment:
Technically a nice-to-have, but PIMS splits these out into by-table configuration files. Might help with keeping this file a more manageable size.

Reply:
Alright, I'll take a look and get that going.